sklearn/svm/_classes.py

```diff
@@ -210,7 +212,7 @@ def __init__(
         penalty="l2",
         loss="squared_hinge",
         *,
-        dual=True,
+        dual="auto",
```

Comment:
For making the change for the future, you can check the following documentation: https://scikit-learn.org/dev/developers/contributing.html#change-the-default-value-of-a-parameter